lib/DBD/mysql.pm

```diff
@@ -1186,6 +1186,18 @@ flags:
 These are used to specify the respective parameters of a call
 to mysql_ssl_set, if mysql_ssl is turned on.
 
+=item mysql_ssl_verify_server_cert
+
+This turns on the MYSQL_OPT_SSL_VERIFY_SERVER_CERT connection option.
+
+Note that this is deprecated as of MySQL 5.7.11 and is removed in MySQL 8.0
+
+=item mysql_ssl_mode
+
+Can be set to 'disabled', 'preferred', 'required', 'verify_ca' or 'verify_identity'.
+Default is 'preferred'.
+
+Available from MySQL 5.7.11
```

Comment:
Now I'm reading about that BACKRONYM and it describe just downgrade attack.

But what I pointed is that if somebody start doing that downgrade attack then you detect that attack **after** you send mysql password over (downgraded) non secure network!

As I wrote, calling `mysql_get_ssl_cipher()` (to detect if somebody is doing downgrade MITM attack) can be done **after** `mysql_real_connect()` which already send password to other side.